envi/interactive.py

```diff
@@ -1,35 +1,78 @@
+STYPE_NONE = 0
```

Comment:
So do you have any docs on how exactly someone is supposed to use this? Because I'm unclear what the right way to use this entire module is.

Reply:
i don't, but that's easy enough.  i'll add something in the module (and docs if you like), but basically... anywhere in the codebase i want (or during runtime):
```
import envi.interactive as ei; ei.dbg_interact(locals(), globals())
```
drops me into an IPython shell to interact with the workspace using IPython.
eventually, i intend to add a menu option to allow an IPython shell (or possibly jupiter?) with the `vw` as a local variable.

although it looks like i want to add an option to spin up a new thread if we want to do it from the GUI without freezing the GUI.
with the current code, that would make this into more of a:
```
import threading
import envi.interactive as ei
t = threading.Thread(target=ei.dbg_interact, args=(locals(), globals()), daemon=True)
t.start()
```

which would make sense to add as optional functionality for `dbg_interact`

Reply:
done